README.md

```diff
@@ -196,19 +196,51 @@ and/or using the [JCasC plugin](https://plugins.jenkins.io/configuration-as-code
 
 If you're unsure which method to use, use JCasC.
 
+### JCasC plugin
+
+Install the [configuration-as-code plugin](https://plugins.jenkins.io/configuration-as-code/) and follow [its example](https://github.com/jenkinsci/configuration-as-code-plugin/tree/master/demos/docker).
+
+As another alternative, a Docker daemon can listen to requests from remote hosts by following the [Docker documentation](https://docs.docker.com/config/daemon/remote-access/).
+The following configuration as code example creates a cloud named "my-docker-cloud" that uses the docker daemon at port 2375 on dockerhost.example.com to run up to 3 containerized agents at a time.
+Agents run the [Jenkins Alpine inbound agent container image](https://hub.docker.com/r/jenkins/inbound-agent) with Java 21.
+Thay use an inbound connection and run as the user ID 1000 with the home directory "/home/jenkins/agent".
+
+```yaml
+jenkins:
+  clouds:
+  - docker:
+      containerCap: 3
+      dockerApi:
+        connectTimeout: 23
+        dockerHost:
+          uri: "tcp://dockerhost.example.com:2375"
+        readTimeout: 43
+      errorDuration: 313
+      name: "my-docker-cloud"
+      templates:
+      - connector:
+          jnlp:
+            jenkinsUrl: "https://jenkins.example.com/"
```

Comment:
FWIW this is the part that gave me trouble during testing using a local Docker daemon: this URL needs to be accessible from the container, and if Jenkins is just running locally on the Docker host (as in `mvn hpi:run`), there is not a standard way to make that happen.